package.json

```diff
@@ -38,6 +38,7 @@
     "mongodb": "^6.5.0",
     "node-fetch": "^3.3.2",
     "p-limit": "^4.0.0",
+    "runestone-lib": "git+https://github.com/me-foundation/runestone-lib.git#d30d6b6",
```

Comment:
why do we need to lock it to this commit?

Reply:
Is not present as npm package so I just took the last version commit https://github.com/me-foundation/runestone-lib/releases/tag/v1.0.2 . 
I really want to control how we handle updates in this library since its heavily used for the runes indexing.

Reply:
Ok ok, I think we can use our own fork. In case they decide to close/hide this repo